core/src/main/java/org/apache/accumulo/core/cli/ClientOpts.java

```diff
@@ -252,8 +182,7 @@ public void updateKerberosCredentials() {
    * overwrite the options if the user has provided something specifically.
    */
   public void updateKerberosCredentials(boolean clientSaslEnabled) {
-    if ((saslEnabled || clientSaslEnabled) && null == tokenClassName) {
-      tokenClassName = KerberosToken.CLASS_NAME;
+    if (saslEnabled || clientSaslEnabled) {
```

Comment:
Why remove the `null == tokenClassName` portion? If the code was changed so that the tokenClassName is never null... then is this block needed at all anymore?

Reply:
I don't think it's needed anymore.  I am going to push a commit that removes it.